AstSemantics.md

```diff
@@ -96,107 +120,61 @@ following rules:
 Wrapping of integers simply discards any upper bits; i.e. wrapping does not
 perform saturation, trap on overflow, etc.
 
-## Addressing local variables
+In addition to storing a value to linear memory, store instructions also return
+their value operand, with no conversion applied.
 
-Each function has a fixed, pre-declared number of local variables which occupy a single
-index space local to the function. Parameters are addressed as local variables. Local
-variables do not have addresses and are not aliased in the globals or memory. Local
-variables have *Local types* and are initialized to the appropriate zero value for their
-type at the beginning of the function, except parameters which are initialized to the values
-of the arguments passed to the function.
+### Addressing
 
-  * `get_local`: read the current value of a local variable
-  * `set_local`: set the current value of a local variable
-
-The details of index space for local variables and their types will be further clarified,
-e.g. whether locals with type `int32` and `int64` must be contiguous and separate from 
-others, etc.
+Each linear memory access operation also has an address operand and an immediate
+integer offset attribute. The infinite-precision sum of the address operand's
```

Comment:
How about "immediate integer byte offset attribute"? I updated the pull request to say that.